f2/src/summary/HowDidItStartSummary.js

```diff
@@ -26,8 +26,8 @@ export const HowDidItStartSummary = props => {
     //Obtain all the array data into the summary array
     howdiditstart.howDidTheyReachYou.map(key =>
       summary.push(
-        key === 'howDidTheyReachYou.others'
-          ? howdiditstart.others
+        key === 'howDidTheyReachYou'
```

Comment:
This will always fail, if this is the desired outcome, we do not need this ternary conditional. 
Instead, I would suggest checking if `key === 'howDidTheyReachYou.others' && howdiditstart.others != ''`